docs/contributing.md

```diff
@@ -1,49 +1,85 @@
 # Developer docs
 
-## Contributing to JAX AI Stack Documentation
+## Contributing to the JAX AI Stack Documentation
+
 The documentation in the `jax-ai-stack` repository is meant to build on documentation
 of individual packages, and specifically cover topics that touch on multiple packages
 in the stack. If you see something missing and would like to contribute, please first
 [open an issue](https://github.com/jax-ml/jax-ai-stack/issues/new) and let us know what
 you have in mind!
 
+## Pre-requisites
+
+To contribute to the documentation, you will need to set your development environment.
+
+You can create a virtual environment and install the dependencies with the following commands:
+
+```bash
+virtualenv jax-docs
+# activate the virtual environment
+source jax-docs/bin/activate
+# install the dependencies
+python -m pip install -r docs/requirements.txt
+```
+
 ## Documentation via Jupyter notebooks
-The jax-ai-stack documentation includes a number of Jupyter notebooks which are rendered
+
+The `jax-ai-stack` documentation includes Jupyter notebooks that are rendered
 directly into the website via the [myst-nb](https://myst-nb.readthedocs.io/) extension.
-In order to ease review and diff of notebooks, we keep markdown versions of the content
+To ease review and diff of notebooks, we keep markdown versions of the content
 synced via [jupytext](https://jupytext.readthedocs.io/).
 
 ### Adding a new notebook
+
+We aim to have one notebook per topic or tutorial covered.
 To add a new notebook to the repository, first move the notebook into the appropriate
-location in the `docs` directory. For example, you could do something like this:
-```
-mv ~/new-tutorial.ipynb docs/new_tutorial.ipynb
-```
-Next, we use jupytext to mark the notebook for syncing with markdown:
+location in the `docs` directory:
+
+```bash
+mv ~/new-tutorial.ipynb jax-ai-stack/docs/new_tutorial.ipynb
```

Comment:
It looks like this command assumes a different root directory than the following commandds. Maybe remove `jax-ai-stack` from the directory path?